pkg/util/kubernetes/hpa/datadogexternal.go

```diff
@@ -9,16 +9,31 @@ package hpa
 
 import (
 	"errors"
+	"expvar"
 	"fmt"
 	"strings"
 	"time"
 
+	"github.com/paulbellamy/ratecounter"
+
 	"gopkg.in/zorkian/go-datadog-api.v2"
 
 	"github.com/DataDog/datadog-agent/pkg/config"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+var (
+	datadogStats          = expvar.NewMap("datadog")
```

Comment:
any chance we can be more specific ? `datadog-backend` or `datadog-external` ? Thinking about when we will have `datadog-agent-calls` or the like.

Reply:
I like `datadog-api`?